packages/graphql/src/schemaTypes/objectTypes/gql-ErrorWrapper.ts

```diff
@@ -0,0 +1,106 @@
+import { stripAnsi, stackUtils, ErrorWrapperSource } from '@packages/errors'
+import { objectType } from 'nexus'
+import path from 'path'
+
+import { ErrorTypeEnum } from '../enumTypes/gql-ErrorTypeEnum'
+import { FileParts } from './gql-FileParts'
+
+export const ErrorWrapper = objectType({
+  name: 'ErrorWrapper',
+  description: 'Base error',
+  definition (t) {
+    t.string('title', {
+      description: 'Optional title of the error. Used to optionally display a title above the error',
+    })
+
+    t.nonNull.string('errorName', {
+      description: 'Name of the error class',
+      resolve (source) {
+        return source.cypressError.originalError?.name || source.cypressError.name
+      },
+    })
+
+    t.nonNull.string('errorStack', {
+      description: 'The error stack of either the original error from the user',
```

Comment:
"either the original error from the user" or what else?

Reply:
Will update